src/ape_cache/query.py

```diff
@@ -0,0 +1,337 @@
+from pathlib import Path
+from typing import Any, Dict, Iterator, List, Optional
+
+import pandas as pd
+from sqlalchemy import create_engine
+from sqlalchemy.engine import CursorResult  # type: ignore
+from sqlalchemy.sql import insert, text
+from sqlalchemy.sql.expression import Insert, TextClause
+
+from ape.api import QueryAPI, QueryType
+from ape.api.networks import LOCAL_NETWORK_NAME
+from ape.api.query import BaseInterfaceModel, BlockQuery, BlockTransactionQuery, ContractEventQuery
+from ape.exceptions import QueryEngineError
+from ape.logging import logger
+from ape.utils import singledispatchmethod  # type: ignore
```

Comment:
This type ignore is not necessary, unless something else is wrong.

Reply:
[ace42d4](https://github.com/ApeWorX/ape/pull/680/commits/ace42d41f8abd354d3f0e938bb8a20679734950e)